### ISSUE TITLE
'Modules in libc++' documentation references closed bugs as blockers

### ISSUE THREAD

**Issue by cjdb** [llvm/llvm-project#75745]
The documentation mentions #61465 and #62943 as potential issues, but these have been closed.

**Comment by mordante** [comment]
Good catch!  @ChuanqiXu9 can you confirm these issues are fully resolved?

**Comment by ChuanqiXu9** [comment]
> Good catch!  @ChuanqiXu9 can you confirm these issues are fully resolved?

#61465 is not closed.  And the other one is closed.
